scripts/git/pre-commit.linter_python

```diff
@@ -43,16 +43,23 @@ if [ "$ans" -ne "0" ]; then
     exit 0
 fi
 
-echo "=================================================="
-echo "  Analysing python code style with 'black'."
-# This check the files but they will not be commited
-black . --include ${files} --check --quiet
-if [ "$?" -eq "1" ]; then
-    echo "Linter checks using 'black' passed."
-else
-    echo "Errors detected with 'black'. Fixing them. Try to add and commit your files again."
-    black . --include ${files}
-    failed_lint=true
+if false; then
+    echo "=================================================="
+    echo "  Analysing python code style with 'black'."
+
+    # Remove 'build' from black's default exclusions. We have
+    # a Python script in a 'build' folder.
+    default_black_exclude="/(\.direnv|\.eggs|\.git|\.hg|\.mypy_cache|\.nox|\.tox|\.venv|venv|\.svn|_build|buck-out|build|dist)/"
```

Comment:
is this repeated on accident? (I don't see it being used

Reply:
No, it's just there to document the black default exclude string so you can see the only change we're making is to remove `build`. It could be commented out if that would be less confusing.